tests/test_task.py

```diff
@@ -0,0 +1,50 @@
+from .conftest import HqEnv
+from .utils import wait_for_job_state
+
+
+def test_task_list_single(hq_env: HqEnv):
+    hq_env.start_server()
+    hq_env.start_worker()
+
+    hq_env.command(
+        ["submit", "--array=0-3", "--", "bash", "-c", "echo $HQ_JOB_ID-$HQ_TASK_ID"]
```

Comment:
The output is not important here. So let's use something simpler, like `hostname`, to not confuse the reader who might think that this complex output is important for the test :)